### ISSUE TITLE
Performance comparison with CLDNN

### ISSUE THREAD

**Issue by MahlerMozart** [intel/mkl-dnn#520]
Hello, 

I am so excited to see MKL-DNN now supports GPU inference in V1.0. I am curious that have you ever compared the performance with CLDNN, especially the inference time? Thanks.

**Comment by karturov** [comment]
Hi Mahler,

Thanks for your interest.

We developed GPU support in MKL-DNN in close cooperation with clDNN team. Due to our experiments, MKL-DNN demonstrate on par or better performance comparing to clDNN on Intel Graphics.

For example, we were able to run and validate accuracy of fp16 inference for ResNet-50 and Inception-V3 models using our internal version of IntelCaffe framework with experimental GPU support. On the Intel Iris Pro Graphics P580 (GT4e) GPUs, for batch sizes 32/64, we saw up to 180 fps on ResNet-50 (similar to clDNN-based inference) and up to 120 fps on Inception-V3 (~1.5 faster than clDNN).

Intel will be adding new optimizations for future CPU and GPU platforms into MKL-DNN, while clDNN library support will be limited to maintaining existing users.

Regards,
Konstantin
